src/Core/Framework/Routing/ApiRouteScope.php

```diff
@@ -12,9 +13,9 @@
 #[Package('framework')]
 class ApiRouteScope extends AbstractRouteScope implements ApiContextRouteScopeDependant
 {
-    final public const ID = 'api';
+    final public const ID = DefinitionService::API;
```

Comment:
same here, especially the ApiRouteScope should not depend on the DefinitionService, those consts can be changed independently, I would keep the string here